### DISCUSSION TITLE
output single chapters in Quarto book

### DISCUSSION THREAD

**Discussion by hueyrb** [quarto-dev/quarto-cli#4553] [Q&A] [answered]
Writing a book in Quarto.  I'd like to send individual chapters to colleagues for review, but not send them the entire book.  I could send everything to a .docx and then delete most chapters, but I wonder whether there is a neater way to do this.  Of course, I could copy the chapter and set up a single chapter document, but that seems inelegant. Any suggestions will be appreciated.

**Comment by jcolomb** [comment, marked as answer]
I would encourage you to think about the review process in general: can they review the .qmd file directly and write comments in markdown document (then you may want to create specific hackmd links for each chapter), if you are happy with only commenting the files, you may do it on a html version of the book, see https://quarto.org/docs/reference/projects/books.html#comments

There seems to be no easy way to get one file per chapter, so manual editing of the _quarto.yml file seems to be the only way to create one chapter at a time (?)

**Comment by hueyrb** [comment]
HIJulien

Thanks for your suggestions!  Manual editing of the _quarto.yml is working.
I appreciate the tip on comments.  I'll give Utterances a try.  I didn't
know about that option.

with best wishes,

Ray
